src/main/resources/celements.hbm.xml

```diff
@@ -52,6 +52,9 @@
     <property name="css_styles" type="string">
       <column name="CEL_CSS_STYLES" />
     </property>
+    <property name="event_data_attr" type="string">
```

Comment:
The HTML5 definition does not give a max-length of an attribute value. HTML4 states 65635. Even though it is not obvious when to use a event-data-attr with that long data, I suggest to use `type="text"` instead. This makes obvious that there is now sensible limit.